### DISCUSSION TITLE
rpgMaker启动全屏后无法改回

### DISCUSSION THREAD

**Discussion by Matokaa** [Blinue/Magpie#851] [Q&A]
rpgmaker启动全屏后无法使用快捷键 但其他类型的视窗都可以

之后开用rpgmaker的游戏全都是全屏 快速键没办法改回视窗

现在不管是用Magpie开还是直接开游戏都是全屏无法做更改  想问有没有办法改回来恢复成语设置

**Comment by Blinue** [comment]
你可能激活了游戏原生全屏，和 Magpie 没关系。你可以看看设置里有没有全屏选项，或者用快捷键，可能是 F4、F11 或 Alt+Enter。

**Comment by Axer128** [comment]
Which rpg maker version? It works well with VX Ace. I use a plain bililinear filter.
